packages/salesforcedx-vscode-core/src/commands/forceSourceRetrieveManifest.ts

```diff
@@ -106,7 +106,7 @@ export async function forceSourceRetrieveManifest(explorerPath: vscode.Uri) {
     sfdxCoreSettings.getBetaDeployRetrieve()
       ? new LibrarySourceRetrieveManifestExecutor()
       : new ForceSourceRetrieveManifestExecutor(),
-    new ConflictDetectionChecker(messages)
+    new TimestampConflictChecker(true, messages)
```

Comment:
Does this need to be gated behind a feature flag?

Reply:
The feature flag is the first thing hit in the check() method that gets called!